### PR TITLE
int-add-m1 Add screen recording access to Reemo in MacOS

### PR DESCRIPTION
**Pull request by tgatignon**
### Your checklist for this pull request

- [X] Check that the commit messages match our requested structure.
- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Add screen recording access to Reemo in MacOS

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: tutorials/access-mac-mini-with-reemo/index.mdx [hunk 1]
```diff
@@ -73,8 +73,15 @@ In this tutorial, you will learn how to launch your Mac mini M1 development envi
 
     </Message>
 4. Enter the **private key** of your Reemo account when prompted. Press Enter to validate.
+5. Reemo needs to have access to screen recording to work. This needs to be done on MacOS GUI, so if you installed via SSH you'll need to connect via VNC. Then, in the System Settings, click Privacy & Security in the sidebar (You may need to scroll down). Then click Screen Recording and turn on screen recording for the Reemo app. 
+
+Click Screen Recording.
+
+Turn screen recording on or off for each app in the list.
```
Comment (RoRoJ) [inline tutorials/access-mac-mini-with-reemo/index.mdx:80]: As above, is this step necessary?

File: tutorials/access-mac-mini-with-reemo/index.mdx [hunk 2]
```diff
@@ -73,8 +73,15 @@ In this tutorial, you will learn how to launch your Mac mini M1 development envi
 
     </Message>
 4. Enter the **private key** of your Reemo account when prompted. Press Enter to validate.
+5. Reemo needs to have access to screen recording to work. This needs to be done on MacOS GUI, so if you installed via SSH you'll need to connect via VNC. Then, in the System Settings, click Privacy & Security in the sidebar (You may need to scroll down). Then click Screen Recording and turn on screen recording for the Reemo app. 
+
+Click Screen Recording.
```
[the remark this answers is not in the document]
Comment (tgatignon) [reply tutorials/access-mac-mini-with-reemo/index.mdx:78]: I'm not sure what you mean. You need to:
- go to system settings
- click on Privacy and Security
- click on screen recording
- turn on screen recording for the Reemo app by clicking the slider
Each click should be clicked once. Slack me if unclear

File: tutorials/access-mac-mini-with-reemo/index.mdx (hunk 1 above)
Comment (tgatignon) [reply tutorials/access-mac-mini-with-reemo/index.mdx:80]: See  comment above

**Review by RoRoJ — APPROVED** [review]

File: tutorials/access-mac-mini-with-reemo/index.mdx [hunk 3]
```diff
@@ -73,8 +73,15 @@ In this tutorial, you will learn how to launch your Mac mini M1 development envi
 
     </Message>
 4. Enter the **private key** of your Reemo account when prompted. Press Enter to validate.
+5. Give Reemo access to screen recording (necessary for it to work). This needs to be done on MacOS GUI, so if you installed via SSH, [connect via VNC](/compute/apple-silicon/how-to/connect-to-mac-mini-m1/#how-to-connect-via-vnc). Then, in **System Settings**, click **Privacy & Security** in the sidebar (you may need to scroll down). Then click **Screen Recording** and turn on screen recording for the Reemo app. 
+
+Click Screen Recording.
```
Comment (RoRoJ) [inline tutorials/access-mac-mini-with-reemo/index.mdx:78]:
```suggestion
```

File: tutorials/access-mac-mini-with-reemo/index.mdx [hunk 4]
```diff
@@ -73,8 +73,15 @@ In this tutorial, you will learn how to launch your Mac mini M1 development envi
...
+
+Turn screen recording on or off for each app in the list.
```
Comment (RoRoJ) [inline tutorials/access-mac-mini-with-reemo/index.mdx:80]:
```suggestion
```

File: tutorials/access-mac-mini-with-reemo/index.mdx [hunk 5]
```diff
@@ -73,8 +73,13 @@ In this tutorial, you will learn how to launch your Mac mini M1 development envi
 
     </Message>
 4. Enter the **private key** of your Reemo account when prompted. Press Enter to validate.
+5. Give Reemo access to screen recording (necessary for it to work). This needs to be done on MacOS GUI, so if you installed via SSH, [connect via VNC](/compute/apple-silicon/how-to/connect-to-mac-mini-m1/#how-to-connect-via-vnc). Then, in **System Settings**, click **Privacy & Security** in the sidebar (you may need to scroll down). Then click **Screen Recording** and turn on screen recording for the Reemo app. 
+
+
```
Comment (RoRoJ) [inline tutorials/access-mac-mini-with-reemo/index.mdx:78]:
```suggestion
```

File: tutorials/access-mac-mini-with-reemo/index.mdx [hunk 6]
```diff
@@ -73,8 +73,13 @@ In this tutorial, you will learn how to launch your Mac mini M1 development envi
...
 
     Your remote computer is now displayed on your Reemo dashboard and ready to be streamed from any Chrome Browser (Desktop, Mobile etc..).
+    
+
```
Comment (RoRoJ) [inline tutorials/access-mac-mini-with-reemo/index.mdx:82]:
```suggestion
```

**Pull request merged**